From 8e16498140bdf7c2bbd916ebb5ce402269fb9513 Mon Sep 17 00:00:00 2001
From: "qinglin.li" <qinglin.li@amlogic.com>
Date: Tue, 18 Apr 2023 20:38:32 +0800
Subject: [PATCH] moddep: ignore unknown symbol error [1/1]

PD#SWPL-118733

Problem:
unknown symbol error

Solution:
ignore unknown symbol error

Verify:
sc2

Change-Id: Ifa432fd421b30403f3a42ea1c9ed69428afbd1dd
Signed-off-by: qinglin.li <qinglin.li@amlogic.com>
---
 build_utils.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build_utils.sh b/build_utils.sh
index ff66ab3..8dd6543 100644
--- a/build_utils.sh
+++ b/build_utils.sh
@@ -71,8 +71,8 @@ function run_depmod() {
     cat ${depmod_stderr} >&2
     if { grep -q "needs unknown symbol" ${depmod_stderr}; }; then
       echo "ERROR: kernel module(s) need unknown symbol(s)" >&2
-      rm -f ${depmod_stderr}
-      exit 1
+      #rm -f ${depmod_stderr}
+      #exit 1
     fi
     rm -f ${depmod_stderr}
   )
-- 
2.25.1

